src/fdc/MSXtar.cc

```diff
@@ -264,13 +264,20 @@ unsigned MSXtar::getNextSector(unsigned sector)
 	}
 }
 
-// get start cluster from a directory entry,
-// also takes care of BAD_FAT and EOF_FAT-range.
-static unsigned getStartCluster(const MSXDirEntry& entry)
+// Get start cluster from a directory entry.
+// BAD FAT and EOF FAT-range is normalized to a single EOF_FAT value.
+unsigned MSXtar::getStartCluster(const MSXDirEntry& entry) const
 {
 	return normalizeFAT(entry.startCluster);
 }
```

Comment:
Technically this could remain a non-member function, right?
But if you want to keep it symmetrical with the new setStartCluster() member function, then that's fine for me.

Reply:
It will need to be a member function for the "Tolerant when reading" patch anyway, so might as well do it now already and keep the two methods consistent.